contracts/extensions/OneTxPayment.sol

```diff
@@ -60,108 +51,74 @@ contract OneTxPayment is ColonyExtension {
   }
 
   /// @notice Completes a colony payment in a single transaction
-  /// @dev Assumes that each entity holds administration and funding roles in the same domain,
-  /// although contract and caller can have the permissions in different domains.
-  /// Payment is taken from root domain, and the caller must have funding permission explicitly in the root domain
+  /// @dev Assumes that each entity holds administration and funding roles in the root domain
   /// @param _permissionDomainId The domainId in which the _contract_ has permissions to add a payment and fund it
   /// @param _childSkillIndex Index of the _permissionDomainId skill.children array to get
   /// @param _callerPermissionDomainId The domainId in which the _caller_ has permissions to add a payment and fund it
   /// @param _callerChildSkillIndex Index of the _callerPermissionDomainId skill.children array to get
   /// @param _workers The addresses of the recipients of the payment
   /// @param _tokens Addresses of the tokens the payments are being made in. 0x00 for Ether.
   /// @param _amounts amounts of the tokens being paid out
-  /// @param _domainId The Id of the domain the payment should be coming from
-  /// @param _skillId The Id of the skill that the payment should be marked with, possibly awarding reputation in this skill.
+  /// @param _domainId The domainId the payment should be coming from
+  /// @param _skillId The skillId that the payment should be marked with, possibly awarding reputation in this skill.
   function makePayment(
-    uint256 _permissionDomainId,
+    uint256 _permissionDomainId, // Unused
```

Comment:
Are we now saying that you can't install this extension anywhere other than root? I'm aware we've removed that functionality from the extension manager, but you can't even install it independently with fewer permissions?

I also am struggling to understand why this has been done here but not in the `makePaymentFundedFromDomain` function.

Reply:
Well, if we are taking funds from the root domain then by definition the extension must have root permissions.